portfolio/src/main/java/com/google/sps/servlets/DeleteServlet.java

```diff
@@ -41,7 +41,7 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
     PreparedQuery results = datastore.prepare(query);
     for (Entity entity : results.asIterable()) {
-        datastore.delete(entity.getKey());
+        // datastore.delete(entity.getKey());
```

Comment:
Are you intentionally disabling single comment delete?

Reply:
Yes, but it was just a quick fix for my initial deploy including comments. I plan to add user-authenticated comment deletion, but I didn't want the delete-all functionality to be openly accessible. Let me know if you want me to uncomment it for now.